src/index.js

```diff
@@ -1,12 +1,15 @@
-import { server, logger } from '@automattic/vip-go';
+var express = require('express');
```

Comment:
@elazzabi 

Woops, I missed one. Gotta have whitespace wrapping the args to be consistent with the rest of our repos.

```suggestion
var express = require( 'express' );
```

Should probably get linting hooked up in CI to this repo.

Reply:
Good catch! And yes, having a CI here makes sense I think. I can work on it part of another PR